src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -846,7 +848,7 @@ private void HandleHostError(Exception exception)
         {
             if (exception == null)
```

Comment:
Move this to use `ArgumentNullException.ThrowIfNull`?

Reply:
The reason I didn't here is this project is `netstandard2.1` from the common build props - didn't want to change it _just_ for this to `net6.0` but if we're okay aligning the projects happy to adjust - thoughts?